moha/api.py

```diff
@@ -43,15 +43,8 @@ def generate_connectivity_matrix(self):
 
             return None, self.connectivity_matrix
 
-        for atom1, atom2, bond in self.connectivity:
-            atom1_name, site1 = get_atom_type(atom1)
-            atom2_name, site2 = get_atom_type(atom2)
-            for pair in [(atom1_name, site1), (atom2_name, site2)]:
-                if pair not in atoms_sites_lst:
-                    atoms_sites_lst.append(pair)
-            if max_site < max(site1, site2):  # finding the max index of site
-                max_site = max(site1, site2)
-        self.n_sites = len(atoms_sites_lst)
```

Comment:
We need this line too, because that what breaks the test.
The suggestion after the line 47 would be:
`self.n_sites = max_site`